tests/program_analysis/test_py_ast_to_cast.py

```diff
@@ -0,0 +1,846 @@
+
+    out_cast = cast.CAST([test_C])
+    to_compare = json.dumps(out_cast.to_json_object(),sort_keys=True,indent=None) 
+
+    assert raw_json == to_compare
+
+
+def test_bool_1():
+    prog_name = "test_bool_1.py"
+    folder = "expression"
+
+    raw_json = '{"nodes": [{"body": [{"body": [{"left": {"node_type": "Var", "source_refs": [{"col_end": 5, "col_start": 4, "node_type": "SourceRef", "row_end": 2, "row_start": 2, "source_file_name": "test_bool_1.py"}], "type": "integer", "val": {"name": "x", "node_type": "Name", "source_refs": [{"col_end": 5, "col_start": 4, "node_type": "SourceRef", "row_end": 2, "row_start": 2, "source_file_name": "test_bool_1.py"}]}}, "node_type": "Assignment", "right": {"boolean": true, "node_type": "Boolean", "source_refs": [{"col_end": 12, "col_start": 8, "node_type": "SourceRef", "row_end": 2, "row_start": 2, "source_file_name": "test_bool_1.py"}]}, "source_refs": [{"col_end": 12, "col_start": 4, "node_type": "SourceRef", "row_end": 2, "row_start": 2, "source_file_name": "test_bool_1.py"}]}, {"body": [{"expr": {"arguments": [{"name": "x", "node_type": "Name", "source_refs": [{"col_end": 15, "col_start": 14, "node_type": "SourceRef", "row_end": 5, "row_start": 5, "source_file_name": "test_bool_1.py"}]}], "func": {"name": "print", "node_type": "Name", "source_refs": [{"col_end": 16, "col_start": 8, "node_type": "SourceRef", "row_end": 5, "row_start": 5, "source_file_name": "test_bool_1.py"}]}, "node_type": "Call", "source_refs": [{"col_end": 16, "col_start": 8, "node_type": "SourceRef", "row_end": 5, "row_start": 5, "source_file_name": "test_bool_1.py"}]}, "node_type": "Expr", "source_refs": [{"col_end": 16, "col_start": 8, "node_type": "SourceRef", "row_end": 5, "row_start": 5, "source_file_name": "test_bool_1.py"}]}], "expr": {"left": {"name": "x", "node_type": "Name", "source_refs": [{"col_end": 8, "col_start": 7, "node_type": "SourceRef", "row_end": 4, "row_start": 4, "source_file_name": "test_bool_1.py"}]}, "node_type": "BinaryOp", "op": "Eq", "right": {"boolean": true, "node_type": "Boolean", "source_refs": [{"col_end": 16, "col_start": 12, "node_type": "SourceRef", "row_end": 4, "row_start": 4, "source_file_name": "test_bool_1.py"}]}, "source_refs": [{"col_end": 16, "col_start": 7, "node_type": "SourceRef", "row_end": 4, "row_start": 4, "source_file_name": "test_bool_1.py"}]}, "node_type": "ModelIf", "orelse": [], "source_refs": [{"col_end": 16, "col_start": 4, "node_type": "SourceRef", "row_end": 5, "row_start": 4, "source_file_name": "test_bool_1.py"}]}], "func_args": [], "name": "main", "node_type": "FunctionDef", "source_refs": [{"col_end": 16, "col_start": 0, "node_type": "SourceRef", "row_end": 5, "row_start": 1, "source_file_name": "test_bool_1.py"}]}], "name": "test_bool_1", "node_type": "Module", "source_refs": [{"col_end": null, "col_start": null, "node_type": "SourceRef", "row_end": 6, "row_start": 1, "source_file_name": "test_bool_1.py"}]}]}'
```

Comment:
@titomeister per my request above let's move all of these raw JSON strings to JSON files. (FYI you can transform what you currently have to JSON files easily by using `json.dump()` since these are already serialized JSON strings).